packages/notifier-seeder/src/notifierErrors.ts

```diff
@@ -0,0 +1,16 @@
+import { InternalError } from "pagopa-interop-models";
+
+const notifierErrorCode = {
+  eventV1ConversionError: "eventV1ConversionError",
+} as const;
+
+export type NotifierErrorCode = keyof typeof notifierErrorCode;
```

Comment:
Why not just:

```suggestion
type NotifierErrorCode = "eventV1ConversionError"
```

We don't need a key-value object mapping for the code string because internal errors are never converted into API problems. For FileManager / QueueManager / other internal error usages we did the same: https://github.com/pagopa/interop-be-monorepo/blob/8389a062e70333b743cb2cfe81406c43ed13bb38/packages/commons/src/file-manager/fileManagerErrors.ts#L3

Reply:
I'm agree with you, key-value mapping is not necessary at the moment.